### PR TITLE
fixing unresolved stuff

### PR DESCRIPTION
**Pull request by benjaoming**
Will comment inline because some IDE auto-edits found their way.

### CODE REVIEW

File: kalite/store/api_resources.py [hunk 1]
```diff
@@ -106,8 +107,8 @@ class Meta:
         queryset = StoreTransactionLog.objects.all()
         resource_name = 'storetransactionlog'
         filtering = {
-            "item": ('exact', ),
-            "user": ('exact', ),
+            "item": ('exact',),
+            "user": ('exact',),
             "reversible": ('exact',),
```
Comment (benjaoming) [inline kalite/store/api_resources.py]: Auto-edit

File: kalite/testing/mixins/browser_mixins.py [hunk 2]
```diff
@@ -430,7 +433,7 @@ def browse_to_random_video(self):
     def browser_get_points(self):
         # The following commented line of code returns an element with blank text,
         # possibly due to a race condition, hence querying the element with js which "just works"
-        #points_elem = self.browser.find_element_by_id("points")
+        # points_elem = self.browser.find_element_by_id("points")
```
Comment (benjaoming) [inline kalite/testing/mixins/browser_mixins.py]: Auto-edit

File: kalite/topic_tools/contentrec_tests/explore_tests.py [hunk 3]
```diff
@@ -34,7 +35,7 @@ def setUp(self):
 		self.user1.set_password(self.PASSWORD)
 		self.user1.save()
 
-		#add one exercise
+		# add one exercise
```
Comment (benjaoming) [inline kalite/topic_tools/contentrec_tests/explore_tests.py]: Auto-edit (and everything else that looks like this)

File: kalite/topic_tools/contentrec_tests/explore_tests.py [hunk 4]
```diff
@@ -43,16 +44,16 @@ def setUp(self):
...
... {'icon_src': '', 'topic_page_url': '/math/pre-algebra', 'kind': 'Topic', 'contains': ['Topic', 'Video', 'Exercise'], 'hide': False, 'description': "No way, this isn't your run of the mill arithmetic. This is Pre-algebra. You're about to play with the professionals. Think of pre-algebra as a runway. You're the airplane and algebra is your sunny vacation destination. Without the runway you're not going anywhere. Seriously, the foundation for all higher mathematics is laid with many of the concepts that we will introduce to you here: negative numbers, absolute value, factors, multiples, decimals, and fractions to name a few. So buckle up and move your seat into the upright position. We're about to take off!", 'parent': 'math', 'node_slug': 'pre-algebra', 'render_type': 'Subject', 'children': ['negatives-absolute-value-pre-alg', 'factors-multiples', 'decimals-pre-alg', 'fractions-pre-alg', 'rates-and-ratios', 'applying-math-reasoning-topic', 'exponents-radicals', 'order-of-operations', 'measurement'], 'available': True, 'path': 'khan/math/pre-algebra/', 'id': 'pre-algebra', 'title': 'Pre-algebra', 'extended_slug': 'math/pre-algebra', 'slug': 'pre-algebra', 'in_knowledge_map': False}}]
```
Comment (benjaoming) [inline kalite/topic_tools/contentrec_tests/explore_tests.py]: Auto-edit

File: kalite/topic_tools/contentrec_tests/next_tests.py [hunk 5]
```diff
@@ -4,7 +4,7 @@
 '''
 
 from kalite.topic_tools.content_recommendation import *
-from kalite.facility.models import FacilityGroup
+from kalite.facility.models import FacilityGroup, Facility
```
Comment (benjaoming) [inline kalite/topic_tools/contentrec_tests/next_tests.py]: Missing import

File: kalite/topic_tools/contentrec_tests/resume_tests.py [hunk 6]
```diff
@@ -5,6 +5,7 @@
 
 from kalite.topic_tools.content_recommendation import *
 from kalite.testing.base import KALiteTestCase
+from kalite.facility.models import Facility
```
Comment (benjaoming) [inline kalite/topic_tools/contentrec_tests/resume_tests.py]: Missing import

File: kalite/topic_tools/contentrec_tests/helper_tests.py [hunk 7]
```diff
@@ -5,10 +5,11 @@
 
 from kalite.topic_tools.content_recommendation import *
 from kalite.testing.base import KALiteTestCase
+from kalite.facility.models import Facility
```
Comment (benjaoming) [inline kalite/topic_tools/contentrec_tests/helper_tests.py]: Missing import

File: kalite/topic_tools/contentrec_tests/explore_tests.py [hunk 8]
```diff
@@ -6,6 +6,7 @@
 from django.test.client import RequestFactory
 from kalite.topic_tools.content_recommendation import *
 from kalite.testing.base import KALiteTestCase
+from kalite.facility.models import Facility
```
Comment (benjaoming) [inline kalite/topic_tools/contentrec_tests/explore_tests.py]: Missing import

File: kalite/testing/mixins/browser_mixins.py [hunk 9]
```diff
@@ -1,6 +1,7 @@
 import json
 import time
 import re
+import logging
```
Comment (benjaoming) [inline kalite/testing/mixins/browser_mixins.py]: Missing import

File: kalite/testing/mixins/browser_mixins.py [hunk 10]
```diff
@@ -16,6 +17,8 @@
 
 from random import choice
 
+from kalite.testing.browser import hacks_for_phantomjs
```
Comment (benjaoming) [inline kalite/testing/mixins/browser_mixins.py]: Missing import

File: kalite/store/api_resources.py [hunk 11]
```diff
@@ -7,6 +7,7 @@
 
 from kalite.shared.api_auth.auth import UserObjectsOnlyAuthorization
 from kalite.facility.api_resources import FacilityUserResource
+from django.http.response import Http404
```
Comment (benjaoming) [inline kalite/store/api_resources.py]: Missing import

File: kalite/store/api_resources.py [hunk 12]
```diff
@@ -75,7 +76,7 @@ def obj_get(self, bundle, **kwargs):
         if storeitem:
             return storeitem
         else:
-            raise NotFound('Test with storeitem_id %s not found' % storeitem_id)
+            raise Http404('Test with storeitem_id %s not found' % storeitem_id)
```
Comment (benjaoming) [inline kalite/store/api_resources.py]: NotFound is........... NOT FOUND :D

File: kalite/testing/mixins/browser_mixins.py (hunk 9 above)
Comment (rtibbles) [inline kalite/testing/mixins/browser_mixins.py]: Is there a reason we're not using ```logging = django.conf.settings.LOG```?

Comment (benjaoming) [inline kalite/testing/mixins/browser_mixins.py]: Quickfixing stuff is the main reason... can you import django.conf here? It seems to be outside of an initialized test env...

Comment (rtibbles) [inline kalite/testing/mixins/browser_mixins.py]: Good point - missed that we were in test country here.

**Comment by rtibbles** [comment]
Looks good. Tests pass. Merging.

**Pull request merged**